package.json

```diff
@@ -98,8 +99,11 @@
     "prop-types": "^15.6.0",
     "react": "^16.8.1",
     "react-ace": "^6.3.2",
+    "react-bootstrap-typeahead": "^3.4.3",
```

Comment:
no idea why this one is not pulled as a dependency, do not want to state it manually now :(
Ideas?

Reply:
It got installed into node_modules for me when I did the following:

1) I removed it from package.json
2) rm -rf node_modules
3) npm install

Reply:
that worked :+1: